registration/include/pcl/registration/impl/default_convergence_criteria.hpp

```diff
@@ -90,11 +96,7 @@ pcl::registration::DefaultConvergenceCriteria<Scalar>::hasConverged ()
   if (fabs (correspondences_cur_mse_ - correspondences_prev_mse_) < mse_threshold_absolute_)
```

Comment:
As far as I can see code in absolute and relative case are identically, so they can be merged:
```
  // 3. The relative sum of Euclidean squared errors is smaller than a user defined threshold
  if ((std::abs (correspondences_cur_mse_ - correspondences_prev_mse_) < mse_threshold_absolute_) || // Absolute
      (std::abs (correspondences_cur_mse_ - correspondences_prev_mse_) / correspondences_prev_mse_ < mse_threshold_relative_))  // Relative
  {
    if (iterations_similar_transforms_ >= max_iterations_similar_transforms_)
    {
      iterations_similar_transforms_ = 0;
      convergence_state_ = CONVERGENCE_CRITERIA_ABS_MSE;
      return (true);
    }
    is_similar = true;
  }
```

Reply:
No, these conditions have different convergence state. `CONVERGENCE_CRITERIA_ABS_MSE` and `CONVERGENCE_CRITERIA_REL_MSE`. If you think these state is logically same, I will combine it as your suggestion.